CLAUDE.md

```
@@ -3,3 +3,7 @@
 ## Task Master AI Instructions
 **Import Task Master's development workflow commands and guidelines, treat as if import is in the main CLAUDE.md file.**
 @./.taskmaster/CLAUDE.md
+
+## Changeset Guidelines
+
+- When creating changesets, remember that it's user-facing, meaning we don't have to get into the specifics of the code, but rather mention what the end-user is getting or fixing from this changeset.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)